internal/builder/profiles.go

```diff
@@ -266,7 +266,7 @@ func GetProfileMultitenant(defaultSA string) string {
 const profileClusterMultitenant = `
 - target:
     kind: Deployment
-    name: "(kustomize-controller|helm-controller|notification-controller|image-reflector-controller|image-automation-controller)"
+    name: "(kustomize-controller|helm-controller|notification-controller|image-reflector-controller|image-automation-controller|source-watcher)"
```

Comment:
I think this change is ok since `source-watcher` Deployment will just not be present in Flux 2.6 :+1: